server/src/optimizer/model_simple_aggregates.rs

```diff
@@ -16,31 +16,27 @@ use std::any::Any;
 use std::fmt::{Display, Formatter};
 use std::sync::Arc;
 
-use crate::catalog::ModelTableMetadata;
-use crate::models;
-use crate::tables::GridExec;
-
-use datafusion::arrow::array::{
-    ArrayRef, BinaryArray, Float32Array, Int32Array, Int64Array, UInt64Array,
-};
+use datafusion::arrow::array::{ArrayRef, BinaryArray, Float32Array, Int64Array, UInt8Array};
 use datafusion::arrow::datatypes::DataType;
 use datafusion::arrow::datatypes::Field;
 use datafusion::arrow::datatypes::Schema;
 use datafusion::arrow::record_batch::RecordBatch;
-
 use datafusion::error::Result;
 use datafusion::physical_optimizer::optimizer::PhysicalOptimizerRule;
 use datafusion::physical_plan::aggregates::AggregateExec;
 use datafusion::physical_plan::expressions::format_state_name;
 use datafusion::physical_plan::expressions::{Avg, Count, Max, Min, Sum};
-use datafusion::physical_plan::repartition::RepartitionExec;
 use datafusion::physical_plan::ColumnarValue;
 use datafusion::physical_plan::ExecutionPlan;
 use datafusion::physical_plan::{Accumulator, AggregateExpr, PhysicalExpr};
 use datafusion::prelude::SessionConfig;
 use datafusion::scalar::ScalarValue;
 use datafusion_expr::AggregateState;
 
+use crate::models;
+use crate::tables::GridExec;
+use crate::types::{TimestampArray, Value, ValueArray};
+
```

Comment:
The files in `optimizer` was purposely not fully refactored as that is a separate task and because this PR had already gotten somewhat large. However, a few minor changes had to be made to make `optimizer` compile again after refactoring `tables.rs` and to make it actually rewrite simple aggregate queries again.

Reply:
Completely fine, it did seem odd that this file was missing so much documentation.